src/base/tests/test_util.py

```diff
@@ -1436,6 +1437,21 @@ def test_SelfPrint(self, value):
         evaluated = safe_eval(value, util.SelfPrintEvalContext(), nocopy=True)
         self.assertEqual(str(evaluated), value)
 
+    @parametrize(
+        [
+            (value,)
+            for value in [
+                "[('company_id', 'in', [*company_ids, False])]",
+                "[('company_id', 'in', [False, *company_ids])]",
```

Comment:
I refactored the tests to use the preprocess approach also in the other tests. That way we ensure it works as expected even when nothing was replaced. It would also cover the case when `ast_unparse is None`